### ISSUE TITLE
Requires double click in ListView

### ISSUE THREAD

**Issue by sercanov** [sbugert/react-native-admob#8]
I don't know if it's a repo or Admob SDK issue but clicking the ad is hard and it doesn't respond for first click. It's a huge blocker. I was using sticky banner with another package, I had 10k impressions and 0,50 CTR and now I have 150k impressions but 0,01 CTR. It's worth look into that.

**Comment by sbugert** [comment]
I tried a minimal example on both android and iOS emulator and I was not able to reproduce the problem.
Are you sure that it's not something else that is capturing the first event in your list? Are you using any gesture APIs?

Please try this example, it renders five banners in a ListView and each of them responds to the first click:
```javascript
'use strict';

var React = require('react-native');
var {
  AppRegistry,
  Component,
  StyleSheet,
  Text,
  View,
  ListView,
} = React;

var Banner = require('react-native-admob').AdMobBanner;

class Example extends Component {

  constructor() {
    super();
    const ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});
    this.state = {
      dataSource: ds.cloneWithRows([
        ['Banner 1', 'banner'],
        ['Banner 2', 'banner'],
        ['Banner 3', 'mediumRectangle'],
        ['Banner 4', 'banner'],
        ['Banner 5', 'banner'],
      ]),
    };
  }

  render() {
    return (
        <ListView
          style={styles.list}
          dataSource={this.state.dataSource}
          renderRow={(rowData) => (
          <View style={styles.container}>
              <Text>{rowData[0]}</Text>
              <Banner
                bannerSize={rowData[1]}
                adUnitID={"ca-app-pub-3940256099942544/2934735716"} />
            </View>
            )}
        />
    );
  }
}

var styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    marginTop: 20,
    backgroundColor: '#8AADAD',
  },
  list: {
    marginTop: 30,
  },
});

AppRegistry.registerComponent('Example', () => Example);
```

**Comment by sercanov** [comment]
I never had this problem with simulator + mouse click. And not always with real devices too, it's kind of tricky to get response from a click. Maybe it is related with click region or AdmobSDK after all. 

Here is my structure 

```javascript
_renderRow 
{
if (adsActive)
banner= ( <AdMobBanner
            style={{alignSelf:'center'}}
            bannerSize={"smartBannerPortrait"}
            adUnitID={ adUnitIDS[0] } /> );

return (
... ...
some codes here
... ...
                <View style={styles.separator}/>
              </View>
            </TouchableHighlight>
            {banner}
          </View> );
}
```

**Comment by sbugert** [comment]
Sorry for the late reply, I was really busy!
Do you have any news on the issue?

So I tested my example on both iOS and Android devices and I was not able to reproduce your problem.
I'm sorry but I can't help you with this issue.

Does it happen for iOS and Android or only one of them? If it happened on both I think this would imply that it's either a problem with your code (maybe not the ListView but something else) or with react-native itself.
Otherwise I guess there has to be the same bug in both my iOS and Android implementation.

**Comment by sercanov** [comment]
hi there @sbugert 

Maybe it's related with my UI but I changed to another module to use only sticky banner. Right now I don't have the same problem.

Actually I realized it because of my Admob revenue. Before using this module I had only one sticky banner, afterwards I switched to this and placed 8 more banners to specific points. My ad view increased like 10k to 100k but click rate was down, so I think that maybe users couldn't click it.

Anyway I'll try reproducing issue with a video soon :)
